packages/template-chakra-storefront/src/pages/cart/index.test.js

```diff
@@ -66,84 +65,98 @@ const mockProductBundleBasket = {
 
 // Set up and clean up
 beforeEach(() => {
-    global.server.use(
-        rest.get('*/customers/:customerId/product-lists', (req, res, ctx) => {
-            return res(ctx.delay(0), ctx.json(mockedCustomerProductLists))
-        }),
-        rest.get('*/products/:productId', (req, res, ctx) => {
-            return res(ctx.delay(0), ctx.json(mockProduct))
-        }),
-        rest.get('*/products', (req, res, ctx) => {
-            return res(ctx.delay(0), ctx.json({data: [mockCartVariant]}))
-        }),
-
-        rest.put('*/baskets/:basketId/shipments/:shipmentId', (req, res, ctx) => {
-            const basket = mockCustomerBaskets.baskets[0]
-            const updatedBasketWithShippingMethod = {
-                ...basket,
-                shipments: [
-                    {
-                        ...basket.shipments[0],
-                        shippingMethod: {
-                            description: 'Order received the next business day',
-                            id: '003',
-                            name: 'Overnight',
-                            price: 29.99
-                        },
-                        shippingAddress: {
-                            address1: '4911  Lincoln Street',
-                            postalCode: '97350',
-                            city: 'IDANHA',
-                            countryCode: 'US',
-                            firstName: 'Ward J',
-                            fullName: 'Ward J Adamek',
-                            id: 'b3e1269a2c1d0ad56694206741',
-                            lastName: 'Adamek',
-                            stateCode: 'OR'
+    prependHandlersToServer([
```

Comment:
Replace api mock with util function to reduce the boilderplate code for better readability